apps/src/aiEvaluation/aiFeedbackApi.ts

```diff
@@ -0,0 +1,39 @@
+import {getAuthenticityToken} from '@cdo/apps/util/AuthenticityTokenStore';
+
+export interface FeedbackData {
+  aiInteractionType: string;
+  aiInteractionId?: number;
```

Comment:
Is this really optional? I think we have to have it when we save the feedback

Reply:
This got a little tangled up... if we require it here, we also need to require a StudentWorkEvaluation and AIResponse to have it.  Which is pretty much all ok, except currently we have a "summarize" function that relies on AI (but we are not using bc we hadn't perfected the output yet).  So, we had some zombie code that wasn't doing anything BUT was causing issues for requiring the ids...

I ended up removing the zombie code for now, but it did raise questions on how we want to address the summary feedback in the (near) future